### PR TITLE
Update for mesos

### PR DESCRIPTION
**Pull request by sarah-vmw**
CHANGES:
apr.spec: add aprver=1 to be recognized as libapr-1 in mesos
apr-util.spec: add apuver=1 for mesos
apache-maven.spec: add a script to set PATH of MAVEN
openjdk.spec: add jre/bin path
subversion.spec: create the spec file
mesos.spec: create the spec file
packages_full.json: add apache-maven, subversion, and mesos
sources_list.sha1: added subversion and mesos

### CODE REVIEW

**Comment by sarah-vmw** [comment]
Merge packages_full.json

File: SPECS/apache-maven/apache-maven.spec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Summary:	Apache Maven
 Name:		apache-maven
 Version:	3.3.3
-Release:	0
+Release:	1
```
Comment (toliaqat) [inline SPECS/apache-maven/apache-maven.spec]: 1%{?dist}

File: SPECS/apache-maven/apache-maven.spec [hunk 2]
```diff
@@ -39,12 +39,19 @@ ant -Dmaven.home=$MAVEN_DIST_DIR
...
+
+echo 'export MAVEN_HOME=/opt/%{name}-%{version}' > %{buildroot}/etc/profile.d/%{name}.sh
+echo 'export PATH=$MAVEN_HOME/bin:$PATH' >> %{buildroot}/etc/profile.d/%{name}.sh
+echo 'export MAVEN_OPTS=-Xms256m' >> %{buildroot}/etc/profile.d/%{name}.sh
 
 %files
 %defattr(-,root,root)
 %{_bindir}/*
 %{_libdir}/*
+/etc/profile.d/%{name}.sh
```
Comment (toliaqat) [inline SPECS/apache-maven/apache-maven.spec]: Use tag for /etc folder.

File: SPECS/subversion/subversion.spec [hunk 3]
```diff
@@ -0,0 +1,55 @@
+Summary:    	The Apache HTTP Server
```
Comment (toliaqat) [inline SPECS/subversion/subversion.spec]: Fix Summary.

**Comment by sarah-vmw** [comment]
Thanks Touseef!
I made changes upon your comments.

File: SPECS/mesos/mesos.spec [hunk 4]
```diff
@@ -0,0 +1,55 @@
+Summary:	Mesos
+Name:		mesos
+Version:	0.22.1
+Release:	1
```
Comment (toliaqat) [inline SPECS/mesos/mesos.spec]: 1%{?dist}

File: SPECS/mesos/mesos.spec [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+License:	Apache
+URL:		http://mesos.apache.org
+Group:		Applications/System
+Vendor:		VMware, Inc.
+Distribution: 	Photon
+Source0:	http://apache.mirrors.lucidnetworks.net//mesos/%{name}-%{version}.tar.gz
```
Comment (toliaqat) [inline SPECS/mesos/mesos.spec]: extra /

**Comment by sarah-vmw** [comment]
Thanks again, Touseef!
I update the mesos.spec.

**Comment by toliaqat** [comment]
Thanks, Merged

File: SPECS/subversion/subversion.spec (hunk 3 above)
Comment (YustasSwamp) [inline SPECS/subversion/subversion.spec]: Subversion file is not found on bintray!

Comment (sarah-vmw) [inline SPECS/subversion/subversion.spec]: I will ask Siju to upload two files ,subversion and mesos to the bintray.

Thanks,
Sarah

**Pull request closed without merging**